ansible/roles/ufw/tasks/main.yml

```diff
@@ -1,9 +1,6 @@
 - name: allow incoming traffic on private interface
   ufw: rule=allow direction=in interface={{ ufw_private_interface }}
 
-- name: allow ssh traffic from control machine
-  ufw: rule=allow from={{ control_machine_ip }}
-
```

Comment:
this seemed redundant with the line above. if its in a private cloud, the control machine should be able to reach it on the private interface. If not, as in monoliths, port 22 should be open anyway for ssh